test/open_mic_test.rb

```diff
@@ -0,0 +1,77 @@
+
+# pry(main)> sal.learn(joke_1)
+#
+# pry(main)> open_mic.repeated_jokes?
+# # => true
+
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/open_mic'
+require './lib/user'
+require './lib/joke'
+
+class OpenMicTest < Minitest::Test
+
+  def test_it_exists
+    open_mic = OpenMic.new({location: "Comedy Works", date: "11-20-18"})
+
+    assert_instance_of OpenMic, open_mic
+  end
+
+  def test_location_and_date
+    open_mic = OpenMic.new({location: "Comedy Works", date: "11-20-18"})
+
+    assert_equal "Comedy Works", open_mic.location
+    assert_equal "11-20-18", open_mic.date
+  end
+
+  def test_no_performers
+    open_mic = OpenMic.new({location: "Comedy Works", date: "11-20-18"})
+
+    assert_equal [], open_mic.performers
+  end
+
+  def test_welcome_performers
+    open_mic = OpenMic.new({location: "Comedy Works", date: "11-20-18"})
+    sal = User.new("Sal")
+    ali = User.new("Ali")
+
+    open_mic.welcome(sal)
+    open_mic.welcome(ali)
+
+    assert_equal [sal, ali], open_mic.performers
+  end
+
+  def test_no_repeated_jokes
+    open_mic = OpenMic.new({location: "Comedy Works", date: "11-20-18"})
+    sal = User.new("Sal")
+    ali = User.new("Ali")
+    open_mic.welcome(sal)
+    open_mic.welcome(ali)
+
+    joke_1 = Joke.new(1, "Why did the strawberry cross the road?", "Because his mother was in a jam.")
+    joke_2 = Joke.new(2, "How do you keep a lion from charging?", "Take away its credit cards.")
+    ali.learn(joke_1)
+    ali.learn(joke_2)
+
+    refute open_mic.repeated_jokes?
+  end
+
+  def test_repeated_jokes
+    open_mic = OpenMic.new({location: "Comedy Works", date: "11-20-18"})
+    sal = User.new("Sal")
+    ali = User.new("Ali")
+    open_mic.welcome(sal)
+    open_mic.welcome(ali)
+
+    joke_1 = Joke.new(1, "Why did the strawberry cross the road?", "Because his mother was in a jam.")
+    joke_2 = Joke.new(2, "How do you keep a lion from charging?", "Take away its credit cards.")
+    ali.learn(joke_1)
+    ali.learn(joke_2)
+
+    sal.learn(joke_1)
+
+    assert open_mic.repeated_jokes?
+  end
+
```

Comment:
Since you created helper methods in your OpenMic class, you should test those methods here as well.